zk/stages/stage_sequence_execute.go

```diff
@@ -87,9 +87,11 @@ func SpawnSequencingStage(
 	// we consider the data stream as verified by the executor so treat it as "safe" and unwind blocks beyond there
 	// if we identify any.  During normal operation this function will simply check and move on without performing
 	// any action.
-	isUnwinding, err := handleBatchEndChecks(batchContext, batchState, executionAt, u)
-	if err != nil || isUnwinding {
-		return err
+	if !batchState.isL1Recovery() {
```

Comment:
let's use !batchState.isAnyRecovery() rather than .isL1Recovery()

Reply:
made the change